plugins/com.google.cloud.tools.eclipse.appengine.whitelist.test/src/com/google/cloud/tools/eclipse/appengine/whitelist/XmlTest.java

```diff
@@ -10,23 +11,36 @@
 import org.junit.Before;
 import org.junit.Test;
 import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.xml.sax.SAXException;
 
 public class XmlTest {
     
-  private DocumentBuilder builder;
+  private Document document;
 	
   @Before
-  public void setUp() throws ParserConfigurationException {
+  public void setUp() throws ParserConfigurationException, IOException, SAXException {
     DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
     factory.setNamespaceAware(true);
-    builder = factory.newDocumentBuilder();
+    DocumentBuilder builder = factory.newDocumentBuilder();
+    document = builder.parse(
+        new File("../com.google.cloud.tools.eclipse.appengine.whitelist/plugin.xml"));
   }
     
   @Test
-  public void testPluginXml() throws Exception {
-    // test fails if malformed
-    Document doc = builder.parse(
-        new File("../com.google.cloud.tools.eclipse.appengine.whitelist/plugin.xml"));
-    Assert.assertEquals("plugin", doc.getDocumentElement().getNodeName());
+  public void testPluginXml() {
+    Assert.assertEquals("plugin", document.getDocumentElement().getNodeName());
+  }
+  
+  @Test
+  public void testLoadExtensionClass()  {
+    Element runElement = (Element) document.getDocumentElement().getElementsByTagName("run").item(0);
+    String className = runElement.getAttribute("class");
+    try {
+      Class.forName(className);
```

Comment:
I like this idea and this works for now, but `Class#forName()` isn't a 1-1 match with `IConfigurationElement#createExecutableExtension()`.  For example, it doesn't test that the class is instantiable with a 0-arg constructor.  And `createExecutableExtension()` allows specifying additional data with the class name like `com.google.cloud.Foo:additional data` that is provided via the `IExecutableExtension` interface.